test/sanitizer_suppressions/ubsan

```diff
@@ -55,7 +55,7 @@ unsigned-integer-overflow:MurmurHash3
 unsigned-integer-overflow:CBlockPolicyEstimator::processBlockTx
 unsigned-integer-overflow:TxConfirmStats::EstimateMedianVal
 unsigned-integer-overflow:prevector.h
-unsigned-integer-overflow:script/interpreter.cpp
+unsigned-integer-overflow:EvalScript
```

Comment:
I assume `stacktop` / `altstacktop` don't work here?

Reply:
Not in this context, because these are runtime suppressions.